website/src/cwkw/theme.css.ts

```diff
@@ -28,4 +27,10 @@ export const themeClass = createTheme(theme, {
     cherokee: `Digohweli, "Charis SIL", "serif", "Arial"`,
     smallCaps: "Charis SIL",
   },
-})
\ No newline at end of file
+})
+
+export const colors = {
+  ...theme.colors,
+  transparent: "transparent",
+  inherit: "inherit",
```

Comment:
You don't need to define these in the colors object because they'll never change. When I said use only theme colors, I should've said "unless they're special values". For example we'll never need to change what `inherit` means but we may want to change what "the text color" means.

Reply:
Yeah, I've realized since that this whole css declaration of themes is unnecessary and I just misunderstood how the themes work when adding it in.